packages/tests/web-test-runner.config.js

```diff
@@ -6,32 +6,28 @@
 
 import {createRequire} from 'module';
 import {playwrightLauncher} from '@web/test-runner-playwright';
-import {fromRollup} from '@web/dev-server-rollup';
 import {createSauceLabsLauncher} from '@web/test-runner-saucelabs';
 import {legacyPlugin} from '@web/dev-server-legacy';
-import {resolveRemap} from './rollup-resolve-remap.js';
-import {prodResolveRemapConfig, devResolveRemapConfig} from './wtr-config.js';
 
 const mode = process.env.MODE || 'dev';
 if (!['dev', 'prod'].includes(mode)) {
   throw new Error(`MODE must be "dev" or "prod", was "${mode}"`);
 }
 
-let resolveRemapConfig;
+let exportCondition = 'development';
 if (mode === 'prod') {
   console.log('Using production builds');
-  resolveRemapConfig = prodResolveRemapConfig;
+  exportCondition = 'default';
 } else {
   console.log('Using development builds');
-  resolveRemapConfig = devResolveRemapConfig;
 }
 
 const browserPresets = {
   // Default set of Playwright browsers to test when running locally.
   local: [
     'chromium', // keep browsers on separate lines
-    'firefox', // to make it easier to comment out
-    'webkit', // individual browsers
+    // 'firefox', // to make it easier to comment out
```

Comment:
Uncomment

Reply:
Yes, I'm going to put all this back in place. I uploaded and made a draft PR to show what I had done so far and talk about the renaming problem I ran into.